app/src/menu.js

```diff
@@ -1,124 +1,135 @@
+          label: i18n.t('Copy'),
         },
-      },
-      {
-        label: 'What\'s New...',
-        click() {
-          electron.shell.openExternal('https://github.com/LiskHQ/lisk-nano/releases');
+        {
+          role: 'paste',
+          label: i18n.t('Paste'),
         },
-      },
-    ],
-  },
-];
+        {
+          role: 'selectall',
+          label: i18n.t('Select all'),
+        },
+      ],
+    },
+    {
+      label: i18n.t('View'),
+      submenu: [
+        {
+          role: 'reload',
+          label: i18n.t('Reload'),
+        },
+        {
+          role: 'togglefullscreen',
+          label: i18n.t('Toggle full screen'),
+        },
+      ],
+    },
+    {
+      label: i18n.t('Window'),
+      submenu: [
+        {
+          role: 'minimize',
+          label: i18n.t('Minimize'),
+        },
+      ],
+    },
+    {
+      label: i18n.t('Help'),
+      submenu: [
+        {
+          label: i18n.t('Lisk Website'),
+          click() {
+            electron.shell.openExternal('https://lisk.io');
+          },
+        },
+        {
+          label: i18n.t('Lisk Chat'),
+          click() {
+            electron.shell.openExternal('https://lisk.chat');
+          },
+        },
+        {
+          label: i18n.t('Lisk Explorer'),
+          click() {
+            electron.shell.openExternal('https://explorer.lisk.io');
+          },
+        },
+        {
+          label: i18n.t('Lisk Forum'),
+          click() {
+            electron.shell.openExternal('https://forum.lisk.io');
+          },
+        },
+        {
+          type: 'separator',
+        },
+        {
+          label: i18n.t('Report Issue...'),
+          click() {
+            electron.shell.openExternal('https://lisk.zendesk.com/hc/en-us/requests/new');
+          },
+        },
+        {
+          label: i18n.t('What\'s New...'),
+          click() {
+            electron.shell.openExternal('https://github.com/LiskHQ/lisk-nano/releases');
+          },
+        },
+      ],
+    },
+  ];
 
-export default (app, copyright) => {
+module.exports = (app, copyright, i18n) => {
+  const template = buildTemplate(i18n);
   if (process.platform === 'darwin') {
     const name = app.getName();
     template.unshift({
       label: name,
       submenu: [
         {
           role: 'about',
-          label: 'About',
+          label: i18n.t('About'),
         },
         {
           role: 'quit',
-          label: 'Quit',
+          label: i18n.t('Quit'),
         },
       ],
     });
   } else {
     template[template.length - 1].submenu.push({
-      label: 'About',
+      label: i18n.t('About'),
       click(item, focusedWindow) {
         if (focusedWindow) {
           const options = {
             buttons: ['OK'],
             icon: `${__dirname}/assets/lisk.png`,
-            message: `Lisk Nano\nVersion ${app.getVersion()}\n${copyright}`,
+            message: `${i18n.t('Lisk Nano\nVersion')} ${app.getVersion()}\n${copyright}`,
```

Comment:
I would prefer the `\n`not to be part of the translation string, it might be confusing to the translators. On the other hand, the version should be part of the translations string. 

 `${i18n.t('Lisk Nano')}\n${i18n.t('Version {{version}}', { version: app.getVersion() })}`